### PR TITLE
added nest_rmu_linear.csh which can be used to create nest rum with l…

### PR DESCRIPTION
**Pull request by AlfaAli**
…inearly increasing e-fold time, more stable nesting.

### CODE REVIEW

File: bin/nest_rmu_linear.csh [hunk 1]
```diff
@@ -0,0 +1,165 @@
+#!/bin/csh
+# Create rmu_nest files with variant e-folding time 
+set echo
+set time = 1
+#
+
+setenv TOOLS /home/sm_alfal/sea/TOPAZ5/NERSC-HYCOM-CICE/hycom/hycom_ALL/hycom_2.2.72_ALL
```
Comment (lbertino) [inline bin/nest_rmu_linear.csh:7]: Excuse my ignorance of our own systems, but how is it done with folders now? Do we use environment variables or do we edit them by hand in each script?

File: bin/nest_rmu_linear.csh [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+/bin/rm -f core 
+touch core
+/bin/rm -f fort.21 fort.21A
+C
+setenv FOR021A fort.21A
+setenv FOR051A fort.51A
+C
+#234567890123456789012345678901234567890123456789012345678901234567890123456789
+cat <<'E-o-D' >! fort.99
+S,N boundaries: 20 grid pts with 3-15 day e-folding time
```
Comment (lbertino) [inline bin/nest_rmu_linear.csh:89]: Is this the place where we put the coordinates of the relaxation zone in the model grid? Maybe a comment would be nice.

**Review by lbertino — APPROVED** [review]
Maybe a few comments for the end, explain why the 3-to-9, 9-to-12 etc. are separated.

File: bin/nest_rmu_linear.csh (hunk 2 above)
Comment (AlfaAli) [reply bin/nest_rmu_linear.csh]: Every relaxation zone can be split into multiple boxes on which different e-folding time can be applied. The box coordinate is determined by if,il,jf,jl. 
I have now updated the ".csh" script where I divide each boundary zone into two boxes, also more comments. By defualt it should work with TP5 grid. I have also the rmu_linear.f to hycom_All.

Comment (AlfaAli) [reply bin/nest_rmu_linear.csh]: @lbertino, if you ok  you can merge and close this request

**Review by lbertino — APPROVED** [review]

**Pull request merged**